src/plugins/intel_cpu/src/emitters/x64/jit_eltwise_emitters.cpp

```diff
@@ -2243,5 +2243,189 @@ void jit_select_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const
         h->vblendmps(vmm_dst | k_mask, vmm_src1, vmm_src0);
     }
 }
+
+/// BITWISE_AND ///
+jit_bitwise_and_emitter::jit_bitwise_and_emitter(x64::jit_generator* host, x64::cpu_isa_t host_isa, const std::shared_ptr<ov::Node>& node, Precision exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {}
+
+jit_bitwise_and_emitter::jit_bitwise_and_emitter(x64::jit_generator* host, x64::cpu_isa_t host_isa, Precision exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {}
+
+size_t jit_bitwise_and_emitter::get_inputs_num() const { return 2; }
+
+std::set<std::vector<element::Type>> jit_bitwise_and_emitter::get_supported_precisions(const std::shared_ptr<ngraph::Node>& node) {
+    return {
+        {element::i8, element::i8},
+        {element::i16, element::i16},
+        {element::i32, element::i32},
+        {element::u8, element::u8},
+        {element::u16, element::u16}
```

Comment:
`u32`?

Reply:
Please, correct, if I'm wrong, but in my understending: only CPU plugin uses the emitter, the plugin doesn't provide `u32`. As result, we can not test the emitter for `u32` precision. As result, it's limited by `i8`, `u8` & `i32`.